js/interface.js

```diff
@@ -894,6 +894,10 @@ var DynamicLists = (function() {
           _this.initializeDetailViewSortable();
         });
     },
+    initDefaultDatasourceProvider () {
```

Comment:
This is ES6 syntax. We need to support IE11.

Reply:
> This is ES6 syntax. We need to support IE11.

Already fixed